v3/src/components/graph/components/chartdots.tsx

```diff
@@ -151,7 +151,7 @@ export const ChartDots = memo(function ChartDots(props: IProps) {
         }
       })
       .style('fill', (anID: string) => {
-        return legendAttrID ? dataConfiguration?.getLegendColorForCase(anID) : defaultPointColor
+        return (legendAttrID && dataConfiguration) ? dataConfiguration.getLegendColorForCase(anID) : defaultPointColor
```

Comment:
Similarly, this should be equivalent:
```suggestion
        return legendAttrID && dataConfiguration?.getLegendColorForCase(anID) ?? defaultPointColor
```